### ISSUE TITLE
Copy files in addition to directories?

### ISSUE THREAD

**Issue by Lusito** [elwin013/parcel-plugin-static-files-copy#34]
I have the following file structure:
- [_locales] *
- [.cache]
- [dist]
- [fonts]
- [icons] *
- [node_modules]
- [src]
- .gitignore
- LICENSE *
- manifest.json *
- package-lock.json
- package.json
- README.md *
- tsconfig.json
- tslint.json

Parcel is configured to generate code into `dist/src` and I want to copy the files marked with `*` into `dist`. The only way I made this work was with this config:

```json
{
    "staticFiles": {
        "staticPath": [{
            "staticPath": ".",
            "staticOutDir": "../"
        }],
        "excludeGlob": ["{.cache,dist,fonts,node_modules,src}/**", "!(manifest).json", ".gitignore"],
        "globOptions": {
            "dot": true
        }
    }
}
```

Couple of notes:
- The `dot: true` was needed in order to exclude node_modules completely. Not sure why. No glob expert, but I would have expected this to match everything inside node_modules.
- Why is the root `staticPath` required to be an array here? If I just specify `staticPath` as string, `staticOutDir` is being ignored.

This setup, however means, that every time my file-tree gets additions, which are *not* supposed to end up in the build, I'll have to manually add them to `excludeGlob`.

I'd like to have the possibility to also specify single files to include rather than just directories.
Any chance for this to happen?

**Comment by brianzelip** [comment]
> I'd like to have the possibility to also specify single files to include rather than just directories.

I'm also interested in specifying single files; it seems natural.

I use this plugin in Vue websites and apps to copy over files that aren't related to the vue app (like humans.txt, a [now page](https://nownownow.com/), etc.)

It lessens the need to add an extra directory (or more) into your project. Also, I've found myself forgetting what's in the static dir until I have to go looking, rather than seeing the files all the time alongside the other files in the root directory via my editor's file system viewer.

I'd love to help out to add this functionality. I have forked the repo and looked at index.js, but I'm not sure where to start. @elwin013 if you're willing to provide some guidance, I'm happy to work on it.
